cmd/statusd/main.go

```diff
@@ -318,6 +190,24 @@ func configureStatusService(flagValue string, nodeConfig *params.NodeConfig) (*p
 	return nodeConfig, nil
 }
 
+func parseConfig(configFiles configFlags, config *params.NodeConfig) bool {
```

Comment:
if no files are specified (false is returned), main will show the version information, since it has no config to start the app. This is a change form previous behaviour, where the app would start with defaults.